projects/gateway/project.yaml

```diff
@@ -0,0 +1,11 @@
+homepage: "https://gateway.envoyproxy.io/"
+main_repo: "https://github.com/envoyproxy/gateway.git"
+primary_contact: "envoy-gateway-security@googlegroups.com"
+auto_ccs :
+  - "tnareddy@confluent.io"
```

Comment:
you probably want to add your own email into here? Otherwise, you won't have access to OSS Fuzz

Reply:
added